doc/sslang-lrm.tex

```diff
@@ -715,7 +716,21 @@ \subsection{Sequences}
 
 
 \subsection{Pattern Matching}
-\fixme{todo}
+\begin{grammar}
+<match> ::=  "match" <expr_blk> "{" (  <pat> "=>" "{" <expr> "}" )$^{+}$ "}"
+\end{grammar}
+
+Pattern matches are a succinct way to handle of a variety of cases for the expression in the match clause, known as the scrutinee. If the scrutinee satisfies the pattern in a match arm, the expression in the match arm is evaluated; otherwise, the expression in that arm is not evaluated.
```

Comment:
Saying "succinct way" implies that there are other ways/primitives to accomplish the same task; there aren't. This also suggests that if there are multiple matching arms, they will all be evaluated, which isn't true either.

Let's write:

> Pattern matches are used to break down algebraic data types. A pattern match consists of a ``scrutinee''---what is being matched on---and a number of ``arms.'' Each arm consists of a pattern that the scrutinee is matched against, and a ``consequence'' expression. The value of the entire pattern match is the consequence of the first matching arm. The types of each consequence must be the same in a pattern match, and the type of the scrutinee must match the type of the each pattern; the overall type of the match expression is that of its consequences.